pkgs/top-level/all-packages.nix

```diff
@@ -16063,9 +16063,12 @@ with pkgs;
   procmail = callPackage ../applications/misc/procmail { };
 
   profanity = callPackage ../applications/networking/instant-messengers/profanity {
-    notifySupport   = config.profanity.notifySupport   or true;
-    traySupport     = config.profanity.traySupport     or true;
-    autoAwaySupport = config.profanity.autoAwaySupport or true;
+    notifySupport       = config.profanity.notifySupport         or true;
+    traySupport         = config.profanity.traySupport           or true;
+    autoAwaySupport     = config.profanity.autoAwaySupport       or true;
+    pgpSupport          = config.profanity.pgpSupport            or true;
+    pythonPluginSupport = config.profanity.pythonPluginSupport   or true;
+    python              = config.profanity.pythonPackage         or python3;
   };
```

Comment:
Can you get rid of these config options. If one wants to change an option one can just override the expression.

Reply:
All of them, or only those I've added?